packages/grafana-ui/src/saga-themes/tokens/core.json

```diff
@@ -0,0 +1,893 @@
+    }
+  },
+  "border": {
+    "borderRadius": {
+      "default": {
+        "value": "{dimension.scale}*0.25",
+        "type": "borderRadius",
+        "description": "Standard Corner Radius used throughout Grafana."
+      },
+      "circle": {
+        "value": "100000",
+        "type": "borderRadius",
+        "description": "used to generate full circle elements"
+      },
+      "pill": {
+        "value": "9999",
+        "type": "borderRadius",
+        "description": "used to generate pill-shaped elements"
+      }
+    }
+  },
+  "opacity": {
+    "0": {
+      "value": "0",
+      "type": "opacity"
+    },
+    "2": {
+      "value": "0.02",
+      "type": "opacity"
+    },
+    "5": {
+      "value": "0.05",
+      "type": "opacity"
+    },
+    "10": {
+      "value": "0.1",
+      "type": "opacity"
+    },
+    "15": {
+      "value": "0.15",
+      "type": "opacity"
+    },
+    "20": {
+      "value": "0.2",
+      "type": "opacity"
+    },
+    "25": {
+      "value": "0.25",
+      "type": "opacity"
+    },
+    "30": {
+      "value": "0.3",
+      "type": "opacity"
+    },
+    "40": {
+      "value": "0.4",
+      "type": "opacity"
+    },
+    "50": {
+      "value": "0.5",
+      "type": "opacity"
+    },
+    "60": {
+      "value": "0.6",
+      "type": "opacity"
+    },
+    "70": {
+      "value": "0.7",
+      "type": "opacity"
+    },
+    "75": {
+      "value": "0.75",
+      "type": "opacity"
+    },
+    "80": {
+      "value": "0.8",
+      "type": "opacity"
+    },
+    "85": {
+      "value": "0.85",
+      "type": "opacity"
+    },
+    "90": {
+      "value": "0.9",
+      "type": "opacity"
+    },
+    "95": {
+      "value": "0.95",
+      "type": "opacity"
+    },
+    "98": {
+      "value": "0.98",
+      "type": "opacity"
+    },
+    "100": {
+      "value": "1",
+      "type": "opacity"
+    }
+  },
+  "zIndex": {
+    "90": {
+      "value": "900",
+      "type": "other",
+      "description": "Used usualy in the activePanel."
+    },
+    "100": {
+      "value": "1000",
+      "type": "other",
+      "description": "Used usualy in the navbarFixed."
+    },
+    "110": {
+      "value": "1010",
+      "type": "other"
+    },
+    "120": {
+      "value": "1020",
+      "type": "other",
+      "description": "Used usualy in the sidemenu."
+    },
+    "130": {
+      "value": "1030",
+      "type": "other",
+      "description": "Used usualy in dropdown and typeahead."
+    },
+    "140": {
+      "value": "1040",
+      "type": "other",
+      "description": "Used usualy in tooltip."
+    },
+    "150": {
+      "value": "1050",
+      "type": "other",
+      "description": "Used usualy in modalBackdrop."
+    },
+    "160": {
+      "value": "1060",
+      "type": "other",
+      "description": "Used usualy in modal."
+    },
+    "170": {
+      "value": "1070",
+      "type": "other",
+      "description": "Used usualy in portal."
+    }
+  },
+  "color": {
+    "blue": {
+      "0": {
```

Comment:
We have a problem in code with these, same with `borderWidth`, `opacity` and any other tokens that are numbers in that we cannot access these in the same way we do tokens that are not numbers. For instance to access a border radius we would do something like `tokens.border.borderRadius.default`, while this color we would have to do `tokens.color.blue[0]`. Because in javacript it is not possible to do `tokens.color.blue.0`. This is why when we did spacing tokens we did things like `spacing.x0` instead of `spacing[0]`. We will need to figure out a proper solution for all the ones that start with numbers like this, maybe the solution is that when we convert these to code we change all of the numbers to something else? Like in token studio it is `tokens.color.blue.0` but in code it is `tokens.color.blue.x0` or `tokens.color.blue.blue0`?

Reply:
I think [Tailwind](https://tailwindcss.com/docs/customizing-colors) is a good show case of using tokens. It doesn't work for all our use cases as they have CSS classes, but for colors they start with 50, could we do the same?

Reply:
@Aires-Grafana it is not only about 0 but any number 😞 like `spacing.25` is also not correct in javascript... So we need to find a solution for all of these